packages/connect-web/src/popup/index.ts

```diff
@@ -5,6 +5,7 @@ import { createDeferred, Deferred, scheduleAction } from '@trezor/utils';
 import { POPUP, IFRAME, UI, CoreEventMessage, IFrameLoaded } from '@trezor/connect/src/events';
 import type { ConnectSettings } from '@trezor/connect/src/types';
 import { getOrigin } from '@trezor/connect/src/utils/urlUtils';
+import { VERSION } from '@trezor/connect/src/data/version';
```

Comment:
just brainstorming. this is core VERSION. maybe we should introduce separate  version field for npm packages? which version we actually want to share?

Reply:
We keep all the main packages versions in sync so it shouldn't be an issue right?